Explorer/Assets/DCL/Passport/Modules/Badges/BadgesDetails_PassportModuleController.cs

```diff
@@ -50,6 +52,7 @@ public BadgesDetails_PassportModuleController(
             badgeInfoController = new BadgeInfo_PassportModuleSubController(badgeInfoModuleView, webRequestController, badgesAPIClient, passportErrorsController);
             badgeDetailsCardsController = new BadgeDetailsCards_PassportModuleSubController(view, webRequestController, badgesCategoriesController, badgeInfoController);
 
+            badgeDetailsCardsController.OnBadgeSelected += (id) => OnBadgeSelected?.Invoke(id);
```

Comment:
every time this event is triggered, a new delegate will be created. It is small allocations, but let's avoid it

Reply:
fixed!